### ISSUE TITLE
v2 addon?

### ISSUE THREAD

**Issue by NullVoxPopuli** [DockYard/ember-composable-helpers#450]
Are folks interested?

V2 conversion would make this library work with vite, as well remove all build time complexity (by having no build time complexity)

**Comment by SergeAstapov** [comment]
+1 for this

@snewcomer @cibernox would you be open for adding more maintainers to this addon so we could keep it updated?

**Comment by tcjr** [comment]
@NullVoxPopuli When you say "remove all build time complexity," do you mean that the v2 version just wouldn't support any `only`/`except` [configuration](https://github.com/DockYard/ember-composable-helpers#configuration) in the consuming app?

**Comment by NullVoxPopuli** [comment]
Correct, it wouldn't be needed, since embroider and ember-auto-import (classic builds) only includes what's used
